single-cell-ebi-gxa.yaml

```diff
@@ -260,6 +264,16 @@ tools:
   - name: alevin
     owner: bgruening
     revisions:
-    - 0a5258e41227
```

Comment:
In general, the .yaml shouldn't include revisions, these are managed at the .lock file automatically.

Reply:
The only reason this was here is because I needed to use a more recent version, and this was statically linked to an older version.

Reply:
No problem, what should happen now is that everyday the .lock file will get the latest hashes for existing tools, and if any, they will get installed. All previous tool versions will remain (so if they were actively used in a workflow for instance, they will still be there).